simple_network_sim/loaders.py

```diff
@@ -142,6 +136,12 @@ def _check_overlap(one, two):
 
 class AgeRange:
     """A helper class for an age range."""
+
+
+    # A marker for no upper limit
+    NO_UPPER = 999999
```

Comment:
I think we need to `assert` that the user never pass an "int" larger than `NO_UPPER` (that's not very likely, of course, but just for completeness). I also suggest using `sys.maxsize` for the size of `NO_UPPER`. Also not a huge deal (thesee are age ranges, after all)

Reply:
I've added tests to ensure that users never pass upper bounds that exceed NO_UPPER, however I actually think it's better to keep NO_UPPER lower - I'm voting for 200. This way we can more easily catch typos in age ranges. For instance, I would rather that `"[50, 660)"` throw an error than accidentally be used (assuming it doesn't overlap anywhere else).